repository-data/application/src/main/resources/hcm-config/hst/configurations/common/pages/location.yaml

```diff
@@ -3,8 +3,8 @@ definitions:
   config:
     /hst:hst/hst:configurations/common/hst:pages/location:
       /main:
-        hst:componentclassname: uk.nhs.digital.common.components.BaseGaContentComponent
-        hst:template: location
+        hst:componentclassname: uk.nhs.digital.common.components.LocationComponent
```

Comment:
Can probably just use BaseGaContentComponent directly (similar to visualhub, cyberalert etc) and remove LocationComponent since LocationComponent doesnt provide any extra functionality.